runtime-light/stdlib/array-functions.h

```diff
@@ -0,0 +1,430 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "runtime-core/runtime-core.h"
+
+constexpr int64_t SORT_REGULAR = 0;
+constexpr int64_t SORT_NUMERIC = 1;
+constexpr int64_t SORT_STRING = 2;
+
+template<class T>
+inline string f$implode(const string &s, const array<T> &a) {
```

Comment:
Template function definitions are implicitly `inline`. So it's useless to explicitly mark them inline.